pkcs11/gkm/test-sexp.c

```diff
@@ -101,6 +133,34 @@ test_parse_key (Test *test, gconstpointer unused)
 	g_assert (ret);
 	g_assert (mpi != NULL);
 	gcry_mpi_release (mpi);
+
+	/* DSA */
```

Comment:
Sure. This is certainly a good candidate for a separate commit. Unfortunately there is much more to do to increase the test coverage into green numbers so I tried my best to improve at least the places I was touching.